pkgs/by-name/ey/eyewitness/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+, cmake
+, xvfb-run
+, firefox-esr
+, geckodriver
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "eye-witness";
+  version = "20230525.1";
+
+  src = fetchFromGitHub {
+    owner = "redsiege";
+    repo = "EyeWitness";
+    rev = "v${version}";
+    hash = "sha256-nSPpPbwqagc5EadQ4AHgLhjQ0kDjmbdcwE/PL5FDL4I=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ] ++ [
+    makeWrapper
+  ];
+
+  buildInputs = with python3Packages; [
+    selenium
+    fuzzywuzzy
+    pyvirtualdisplay
+    pylev
+    netaddr
+    pydevtool
+  ] ++ [
+    firefox-esr
+    xvfb-run
+    geckodriver
+  ];
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/eyewitness}
+    cp -R * $out/share/eyewitness
+  '';
+
+  fixupPhase = ''
+    makeWrapper "${python3Packages.python.interpreter}" "$out/bin/eyewitness" \
+      --set PYTHONPATH "$PYTHONPATH" \
+      --add-flags "$out/share/eyewitness/Python/EyeWitness.py"
+  '';
+
+  meta = with lib; {
+    description = "EyeWitness is designed to take screenshots of websites, provide some server header info, and identify default credentials if possible";
```

```suggestion
    description = "Take screenshots of websites, provide some server header info, and identify default credentials if possible";
```

https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#description-var-meta-description

Please also shorten the description if possible.